docs/performance/tune-performance.md

```diff
@@ -51,6 +51,19 @@ In both cases, you will get a JSON file which contains the detailed performance
 * Type chrome://tracing in the address bar
 * Load the generated JSON file
 
+For Cuda EP, performance numbers from device will be attached to those from host. For example:
```

Comment:
Please capitalize CUDA here and in other updated text as well

Reply:
Done.